Source/GSAutoLayoutVFLParser.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (C) 2022 Free Software Foundation, Inc.
+   
+   By: Benjamin Johnson
+   Date: 11-11-2022
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import <AppKit/AppKit.h>
```

Comment:
Within the library itself we prefer to include only the headers that are actually needed, In this case Foundation, NSView and NSLayoutConstraint.

Reply:
Addressed in 1aff56cb9b355fab3d3a95f2473cafd97e69633e